src/pages/settings/Payments/PaymentsPage/paymentsPagePropTypes.js

```diff
@@ -29,6 +31,12 @@ const propTypes = {
     ...withLocalizePropTypes,
 
     ...windowDimensionsPropTypes,
+
+    /** List of bank accounts */
+    bankAccountList: PropTypes.objectOf(bankAccountPropTypes),
+
+    /** List of cards */
+    cardList: PropTypes.objectOf(cardPropTypes),
```

Comment:
NAB; I think we try to keep `...withLocalizePropTypes,` and `...windowDimensionsPropTypes,` at the bottom (did a quick scan and only saw one instance of them being at the top of `propTypes` instead of at the bottom)